tests/test.bats

```diff
@@ -340,3 +340,40 @@ _healthcheck_wait ()
 	### Cleanup ###
 	docker rm -vf "$NAME" >/dev/null 2>&1 || true
 }
+
+@test "Custom Cron Integration" {
+	[[ $SKIP == 1 ]] && skip
+
+	### Setup ###
+	docker rm -vf "$NAME" >/dev/null 2>&1 || true
+	docker run --name "$NAME" -d \
+		-v /home/docker \
+		-v $(pwd)/../tests:/var/www \
+		"$IMAGE"
+	_healthcheck_wait
+
+	### Tests ###
+	# Confirm output from cron is working
+
+	# Sleep for 60 Seconds so cron can run first time.
+	sleep 60
```

Comment:
Ok in this case you should definitely just `echo "The current date is $(date)" > /tmp/date.txt; chmod 0777 /tmp/date.txt` right here